dataprofiler/profilers/profile_builder.py

```diff
@@ -683,3 +687,127 @@ def tqdm(l):
+        """
+        # Delete data labeler for profiler
+        data_labeler_options = self.options.structured_options.data_labeler
+        if data_labeler_options.is_enabled \
+                and data_labeler_options.data_labeler_object is not None:
+                    data_labeler_options.data_labeler_object = None
+                
+        # Delete data labelers for all columns
+        for key in self._profile:
+
+            val = self._profile[key]
+            use_data_labeler = True
+            if val.options and isinstance(val.options, StructuredOptions):
+                use_data_labeler = val.options.data_labeler.is_enabled
+
+            if use_data_labeler:
+                val.profiles['data_label_profile']._profiles['data_labeler'] \
+                   .data_labeler = None
+
+    def _restore_data_labelers(self):
+        """
+        Helper method for restoring all data labelers after saving to or 
+        loading from disk.
+        """
+        # Restore data labeler for profiler
+        data_labeler_options = self.options.structured_options.data_labeler
+        if data_labeler_options.is_enabled \
+                and data_labeler_options.data_labeler_object is None:
+            try:
+                data_labeler = DataLabeler(
+                    labeler_type='structured',
+                    dirpath=data_labeler_options.data_labeler_dirpath,
+                    load_options=None)
+                self.options.set({'data_labeler.data_labeler_object': data_labeler})
+                
+            except Exception as e:
+                utils.warn_on_profile('data_labeler', e)
+                self.options.set({'data_labeler.is_enabled': False})
+                
+        # Restore data labelers for all columns
+        for key in self._profile:
+
+            val = self._profile[key]
+            use_data_labeler = True
+            if val.options and isinstance(val.options, StructuredOptions):
+                use_data_labeler = val.options.data_labeler.is_enabled
+
+            if use_data_labeler:
+                data_labeler_profile = val.profiles['data_label_profile']._profiles['data_labeler']
+                data_labeler_profile.data_labeler = None
+                
+                if val.options and val.options.data_labeler.data_labeler_object:
+                    data_labeler_profile.data_labeler = val.options.data_labeler.data_labeler_object
+                if data_labeler_profile.data_labeler is None:
+                    data_labeler_dirpath = None
+                    if val.options:
+                        data_labeler_dirpath = val.options.data_labeler.data_labeler_dirpath
+                
+                    data_labeler_profile.data_labeler = DataLabeler(
+                        labeler_type='structured',
+                        dirpath=data_labeler_dirpath,
+                        load_options=None)
+
+
+    def save(self, filepath="profile.pkl"):
```

Comment:
based on how the DataLabeler components work, I would expect the api mechanism for loading/saving a profile to be using the class itself, i.e.:
```python
profiler = dp.Profiler(...)
profiler.save(filepath)
.
.
.
# now need to load 
profiler = dp.Profiler.load(filepath)

Reply:
agree

Reply:
That makes sense, I will update the load to make it a class method

Reply:
Changed this in new commit, but I did have to start by making an empty profiler object:
`profile = Profiler(pd.DataFrame([]))`

Reply:
I think the user should be explicit about what they are loading, hence no default value. @lettergram ?